packages/x-charts-pro/src/index.ts

```diff
@@ -28,6 +28,7 @@ export * from '@mui/x-charts/ChartDataProvider';
 export * from '@mui/x-charts/ChartsLabel';
 
 // Pro components
+export { useHeatmapSeries } from './hooks';
```

Comment:
In the community package we're exporting everything from the `hooks` directory. 

I've opted to only export the `useHeatmapSeries` hook for now since I don't know if all hooks from the `hooks` directory are supposed to be used by consumers of the library: are they? If so, I'll change this to export everything from `./hooks`.

Reply:
So currently it is a bit messy. We've discussed about cleaning up and documenting all the hooks. We have an issue for it as well https://github.com/mui/mui-x/issues/16105

In my view, the hooks in the `/hooks/` folder should all be public, but they are currently a mix of internal and public ones. `internal` ones should be moved either to their respective feature or to the internal folder

Reply:
I see. So I'll just export `useHeatmapSeries` for now. Thanks!

Reply:
For info only the first depth of mui package are available.


So the `hook` folder could be used to store both private and public ones. And it's the `/hooks/index.ts` that defines which hook is available for users.

Which simplifies the `x-charts-pro/index.ts` that I guess should always look like an `export * from './theFolderName'`

Because every thing in `'./theFolderName'` is already available with `@mui/x-charts-pro/theFolderName`

Reply:
So does that mean that I should do `export * from './hooks'` in `src/index.ts`? It would mean we would start exporting `useIsZoomInteracting`. Is that fine?

Reply:
Yes, I'm fine with having `isZoomInteracting` open. I think the `export * from './hooks'` just got forgotten

Reply:
Done 👍